diff --git a/build.zig b/build.zig
index de32bd2..c2883b1 100644
--- a/build.zig
+++ b/build.zig
@@ -8,21 +8,7 @@ const raylibFlags = &[_][]const u8{
 };
 
 fn addRayLib(exe: *std.build.LibExeObjStep) void {
-    exe.addIncludeDir("raylib/include");
-    exe.addLibPath("raylib/lib");
-    
-    exe.addIncludeDir("./lib/raylib/src");
-    exe.addIncludeDir("./lib/raylib/src/external/glfw/include");
-    
-    exe.addCSourceFile("./lib/raylib/src/rcore.c", raylibFlags);
-    exe.addCSourceFile("./lib/raylib/src/rmodels.c", raylibFlags);
-    exe.addCSourceFile("./lib/raylib/src/raudio.c", raylibFlags);
-    exe.addCSourceFile("./lib/raylib/src/rshapes.c", raylibFlags);
-    exe.addCSourceFile("./lib/raylib/src/rtext.c", raylibFlags);
-    exe.addCSourceFile("./lib/raylib/src/rtextures.c", raylibFlags);
-    exe.addCSourceFile("./lib/raylib/src/utils.c", raylibFlags);
-    exe.addCSourceFile("./lib/raylib/src/rglfw.c", raylibFlags);
-
+    exe.linkSystemLibrary("raylib");
     exe.linkSystemLibrary("GL");
     exe.linkSystemLibrary("rt");
     exe.linkSystemLibrary("dl");
