diff --git a/linux/CMakeLists.txt b/linux/CMakeLists.txt
index 1b375af..bd356b4 100644
--- a/linux/CMakeLists.txt
+++ b/linux/CMakeLists.txt
@@ -13,13 +13,13 @@ if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
   # We can't really ask users to use a cmake that recent, so there's this if here.
   FetchContent_Declare(
     sqlite3
-    URL https://sqlite.org/2024/sqlite-autoconf-3460000.tar.gz
+    URL @sqlite_src@
     DOWNLOAD_EXTRACT_TIMESTAMP NEW
   )
 else()
   FetchContent_Declare(
     sqlite3
-    URL https://sqlite.org/2024/sqlite-autoconf-3460000.tar.gz
+    URL @sqlite_src@
   )
 endif()
 FetchContent_MakeAvailable(sqlite3)
